CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The gem now supports Ruby 3.0+ and Rails 6.1+. ([#1521], [#1522], [#1547], [#154
 
 ### Features
 * Add normalize matcher by @stephannv. ([#1558])
+* Add validates_comparison_of matcher by @matsales28. ([#1552])
```

Comment:
I noticed that we forgot to add this feature on the changelog

Reply:
Ouch, nice catch! Thank you!